Source/Workers/decodeDraco.js

```diff
@@ -0,0 +1,119 @@
+define([
+        '../Core/ComponentDatatype',
+        '../Core/defined',
+        '../Core/IndexDatatype',
+        '../Core/RuntimeError',
+        '../ThirdParty/draco-decoder-gltf',
```

Comment:
@fanzhanggoogle any chance you are able to quick a quick look at Cesium's use of the Draco API in this file and let us know if you have any feedback or tips?  Thanks!